### DISCUSSION TITLE
Hello. Wakatime user not found

### DISCUSSION THREAD

**Discussion by mohamedmehdigara** [anuraghazra/github-readme-stats#1825] [Q&A] [answered]
[![willianrod's wakatime stats](https://github-readme-stats.vercel.app/api/wakatime?username=mohamedmehdigara)](https://github.com/anuraghazra/github-readme-stats)

Knowing that my username field is exactly the same username in wakatime website.
I confused name with username. Problem solved. Thanks

**Comment by rickstaa** [comment, marked as answer]
@mohamedmehdigara Great to hear that you solved your problem! I noticed your waketime is empty. If you did some coding and it does not show up, you can check out https://github.com/anuraghazra/github-readme-stats/discussions/1793.

**Comment by mohamedmehdigara** [comment]
@rickstaa Thanks. But I still have an empty waketime even after adding  &Cache-Control: no-cache in [![Your Repository's Stats](https://github-readme-stats.vercel.app/api/wakatime?username=mohamedmehdigara&theme=radical&Cache-Control=no-cache)](https://github.com/mohamedmehdigara/mohamedmehdigara) readme code

**Reply by rickstaa** [reply]
@mohamedmehdigara Ah, I see. In that case, I think the WakaTime card is empty because it can not fetch any public coding data from your profile (see https://wakatime.com/@mohamedmehdigara). Please make sure your profile is fully public (see https://github.com/anuraghazra/github-readme-stats/discussions/1793#discussioncomment-2839986).

**Reply by rickstaa** [reply]
@mohamedmehdigara Did that solve your question?
